docs/docs/llms/responses-agent-intro/index.mdx

```diff
@@ -0,0 +1,310 @@
+import { APILink } from "@site/src/components/APILink";
+
+# Tutorial: ResponsesAgent Introduction
+
+## What is a ResponsesAgent?
+`ResponsesAgent` is a subclass of `PythonModel` that provides a framework-agnostic way to create an agent model. Authoring a `ResponsesAgent` provides the following benefits:
+- Ensure your model is compatible with OpenAI Responses API (Read more about it in the [Responses API Introduction](./responses-api-intro))
```

Comment:
I would emphasize the other bullets first - I think the benefit to call out here is that you can author an agent that is compatible with standard clients like the OAI responses client, simplifying writing downstream UIs or applications